pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceTransaction.java

```diff
@@ -0,0 +1,613 @@
+                required = true)
+        public List<String> producerTopic = Collections.singletonList("test");
+
+        @Parameter(names = {"-threads", "--num-test-threads"}, description = "Number of test threads")
+        public int numTestThreads = 1;
+
+
+        @Parameter(names = {"--separator"}, description = "Separator between the topic and topic number")
+        public String separator = "-";
+
+        @Parameter(names = {"-au", "--admin-url"}, description = "Pulsar Admin URL")
+        public String adminURL;
+
+
+        @Parameter(names = {"-u", "--service-url"}, description = "Pulsar Service URL")
+        public String serviceURL;
+
+
+        @Parameter(names = {"-np",
+                "--partitions"}, description = "Create partitioned topics with the given number of partitions, set 0 "
+                + "to not try to create the topic")
+        public Integer partitions = null;
+
+        @Parameter(names = {"-c",
+                "--max-connections"}, description = "Max number of TCP connections to a single broker")
+        public int maxConnections = 100;
+
+        @Parameter(names = {"-time",
+                "--test-duration"}, description = "Test duration in secs. If 0, it will keep publishing")
+        public long testTime = 0;
+
+
+        @Parameter(names = {"-ioThreads", "--num-io-threads"}, description = "Set the number of threads to be " +
+                "used for handling connections to brokers, default is 1 thread")
+        public int ioThreads = 1;
+
+        @Parameter(names = {"-ss",
+                "--subscriptions"}, description = "A list of subscriptions to consume on (e.g. sub1,sub2)")
+        public List<String> subscriptions = Collections.singletonList("sub");
+
+        @Parameter(names = {"-ns", "--num-subscriptions"}, description = "Number of subscriptions (per topic)")
+        public int numSubscriptions = 1;
+
+
+        @Parameter(names = {"-sp", "--subscription-position"}, description = "Subscription position")
+        private SubscriptionInitialPosition subscriptionInitialPosition = SubscriptionInitialPosition.Latest;
+
+
+        @Parameter(names = {"-st", "--subscription-type"}, description = "Subscription type")
+        public SubscriptionType subscriptionType = SubscriptionType.Exclusive;
+
+        @Parameter(names = {"-q", "--receiver-queue-size"}, description = "Size of the receiver queue")
+        public int receiverQueueSize = 1000;
+
+
+        @Parameter(names = {"-timeout", "--txn-timeout"}, description = "transaction timeout")
+        public long transactionTimeout = 5;
+
+        @Parameter(names = {"-nmp", "--numMessage-perTransaction-produce"},
+                description = "the number of messages produced by  a transaction")
+        public int numMessagesProducedPerTransaction = 1;
+        @Parameter(names = {"-nmc", "--numMessage-perTransaction-consume"},
```

Comment:
The number of messages per transaction acknowledgment. (Only --txn-enable true can it take effect)